nistats/reporting/reporting_examples/report_flm_adhd_dmn/report_flm_adhd_dmn.py

```diff
@@ -0,0 +1,51 @@
+import numpy as np
```

Comment:
I wrote this as tests for my use as I developed. I think since these are based on existing Nistats examples, we can add calls to generate reports there, or we can write a fresh one to show this. Which one is better according to you?

Reply:
I would simply add the report geenration to existing examples;